<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RemoveFromStaff</class>
 <widget class="QDialog" name="RemoveFromStaff">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>166</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QComboBox" name="removeList">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="phoneReport">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Phone:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="removeBtn">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>10</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Remove</string>
   </property>
  </widget>
  <widget class="QLabel" name="idReport">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>ID: </string>
   </property>
  </widget>
  <widget class="QLabel" name="wageReport">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Wage:</string>
   </property>
  </widget>
  <widget class="QLabel" name="assignedReport">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Working: </string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
